var/spack/repos/builtin/packages/m4/package.py

```diff
@@ -49,7 +49,10 @@ def configure_args(self):
         args = ['--enable-c++']
 
         if spec.satisfies('%clang') and not spec.satisfies('platform=darwin'):
-            args.append('CFLAGS=-rtlib=compiler-rt')
+            cflags.append('-rtlib=compiler-rt')
```

Comment:
I know I've been away from Spack for a while, so perhaps things have changed internally, but... where is `cflags` defined?

Reply:
Yes, that's precisely what I asked in https://github.com/LLNL/spack/pull/5728#discussion_r144389378.

Reply:
I screwed up, I looked at `cp2k` and forgot to add:
```
cflags = copy.deepcopy(optflags[self.spec.compiler.name])
```
plus
```
args.append('CFLAGS = {0}'.format(' '.join(cflags)))
```
Lame....